### DISCUSSION TITLE
Shiny SQLite Logging Question

### DISCUSSION THREAD

**Discussion by RobSchein** [shinyorg/shiny#1512] [Q&A] [answered]
Hi all.  I am using the BLE features in Shiny, and I also implemented SQLite logging through the Shiny interface.  I am using the same db filename as in the sample code, and I see that not only are my messages being logged, but some BLE messages (like: Startup task 'Shiny.BluetoothLE.BLEManager' ran successfully) are also being logged.  I would presume that I could change the DB name and eliminate those messages, since my initial thought  is that I wouldn't want my logs to be poluted, so to speak, with Shiny logs.  Maybe not, though.  Maybe I want to expose those messages, as well, for sanity sake.  If I change file names or take other corrective actions, I would want to trim any DB files from time to time, as well.  I wanted your take on this topic.

**Comment by aritchie** [comment]
Microsoft.Extensions.Logging is used in shiny.  You can disable any portions of the logging you want based on the configuration of the logging.  Shiny only logs on debug & info levels.  

https://learn.microsoft.com/en-us/dotnet/core/extensions/logging

**Reply by RobSchein** [reply]
Ok, thanks.  Are you saying I can disable Shiny-generated logging by configuration (and if so, how), or disable debug/info levels if I don't want to see Shiny generated messages?

**Reply by aritchie** [reply, marked as answer]
In your mauiprogram.cs

```csharp
builder.Logging.AddFilter("Shiny", LogLevel.Error);
builder.Logging.SetMinimumLevel(LogLevel.Error);
```

**Reply by RobSchein** [reply]
Thank you, sir!
